C/services/common/delivery_queue.cpp

```diff
@@ -439,6 +439,33 @@ void DeliveryQueue::processDelivery(DeliveryQueueElement* elem)
 					   elem->getData()->getNotificationName(),
 					   elem->getData()->getReason(),
 					   elem->getData()->getMessage());
+
```

Comment:
The Jira is about creating asset tracker data for assets that are created by notification delivery. Not for notifications in general. This is the wrong place to have done this as you do not know what the delivery plugin is going to do. This has to be done in the plugin.

Reply:
Hi Mark,
There is a method ingestData() in the DeliveryPlugin class which tells us about the nature of plugin.  We can utilize information from ingestData() method to take decision if plugin will ingest some asset or not  into database. Does it make sense ?